packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -83,24 +86,31 @@ const initAppState = async ({App, component, match, route, req, res, location})
 
     const {params} = match
 
+    const queryCache = queryClient.getQueryCache()
+    const queries = queryCache.getAll()
+    const queryPromises = queries.map(({fetch, enabled}) => enabled && fetch())
```

Comment:
Thanks for getting this fixed quickly.
To recap a conversation that @bendvc and I have earlier, I raised a concern how the previous version `const queries = queryClient.query cache.queries.map((q) => q.fetch())` worked when the users do a dependent query calls. (e.g [here](https://tanstack.com/query/v4/docs/guides/dependent-queries)).  
In other words, it is possible that we can tell useQuery to not fetch until a  piece of information is available, and that is not possible previously. Hence it is necessary that we take the `enabled` into consideration here.

Reply:
Thanks Alex. I have gon ahead and updated the code to 1. not fetch queries that aren't enabled in order to not bloat the html response and 2. use the react query api methods to get those queries and not accessing the possibly private members like **queryCache** and **queries**.